nautobot/core/testing/api.py

```diff
@@ -191,6 +191,37 @@ def test_options_object(self):
             response = self.client.options(url, **self.header)
             self.assertHttpStatus(response, status.HTTP_200_OK)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+        def test_options_object_detail_view_config(self):
```

Comment:
Feels like this and the above `test_options_object` could be merged into a single test?